### DISCUSSION TITLE
Question related to winget's --override flag

### DISCUSSION THREAD

**Discussion by felixsanz** [microsoft/winget-pkgs#131706] [Q&A] [answered]
Maybe every package is different but when installing `Nvidia.CUDA` it installs a lot of things like drivers, documentation, compiler, runtime, a lot of tools... how can I use `--override` here to only install the runtime libs?

I have seen [this answer](https://github.com/microsoft/winget-cli/issues/415#issuecomment-651957316) that solves it for vscode, using the [code.iss file](https://github.com/microsoft/vscode/blob/8c6e828a8e22cef41f12b35f07a2e6e20b15847d/build/win32/code.iss#L76-L83), but I haven't found such file for `Nvidia.CUDA`, when installing the package it downloads an .exe file so I dunno if it's even possible...

**Comment by mdanish-kh** [comment, marked as answer]
That depends on whether the installer supports switches that control what gets installed. If such switches are made available by the publisher, then you can pass those using the `--override` arg.
